transit_odp/avl/signals.py

```diff
@@ -7,4 +7,4 @@
 @receiver(post_delete, sender=PostPublishingCheckReport)
 def post_delete_ppc_report(sender, instance, **kwargs):
     if instance.file.name:
-        instance.file.storage.delete(instance.file.name)
+        instance.file.delete(instance.file.name)
```

Comment:
When you do this on your local database, this deletes the PPC report in `transit_odp/media` does it?

Reply:
YEP, took an age to prove but here it is!
```
/opt/project/transit_odp/media $ ls M*
MyFile.pdf
/opt/project/transit_odp/media $ ls M*
ls: M*: No such file or directory
```